using/settings/pronouns.md

```diff
@@ -1,15 +1,15 @@
 # Pronouns
 
 In Grammar, pronouns are words that can be substituted for nouns.
-For example, if Exercism's co-founder, Jeremy, was to declare my love for Exercism, rather than saying "**Jeremy** loves **Exercism**", Jeremy might say "**I** love **you**".
+For example, if Exercism's co-founder, Jeremy, was to declare their love for Exercism, rather than saying "**Jeremy** loves **Exercism**", Jeremy might say "**I** love **you**".
 Both "I" and "you" are pronouns, that replace the nouns "Jeremy" and "Exercism".
 
 Many languages have gendered pronouns.
-For example, as Jeremy's gender is male, we can say **he** loves Exercism.
-If Jeremy was female, we could say **she** loves Exercism.
+For example, Jeremy is a man and prefers masculine pronouns, so, we might say **he** loves Exercism.
+If Jeremy was a woman, we might say **she** loves Exercism.
 
-Some people don't like others to know their gender.
-There are many reasons for this, including people not wanting to be discriminated against based on their gender, or highly valuing their privacy.
+Some people prefer not to use masculine or feminine pronouns on Exercism.
+There are many reasons for this: some people are non-binary or genderqueer; some want to avoid discrimination based on their gender; some highly value their privacy.
```

Comment:
```suggestion
There are many reasons for this, including: some people may want to avoid discrimination based on their gender; some are non-binary or genderqueer; some highly value their privacy.
```